quality/BUILD.bazel

```diff
@@ -120,6 +121,22 @@ buildifier_test(
     workspace = "//:WORKSPACE",
 )
 
+# TODO(cfrantz): Find a way to keep this list of rust targets synchronized
```

Comment:
Maybe something like a shell-script that checks a query of rust dependencies of this rule matches the list of rust files in the repo? It would also have the effect of monitoring for dead code.